xml/schema/CRM/Civicase/CaseCategoryInstance.xml

```diff
@@ -36,6 +36,13 @@
       <optionGroupName>case_category_instance_type</optionGroupName>
     </pseudoconstant>
   </field>
+  <field>
+    <name>singular_label</name>
+    <type>varchar</type>
+    <length>255</length>
+    <comment>A singular noun for the case category instance</comment>
+    <required>true</required>
+  </field>
```

Comment:
I don't think this is the right place to add this field. The CaseCategoryInstanceDAO mainly links two objects together, i.e the category object and the instance object. Any information regarding the category should be stored on the category table. Unfortunately, the category table is an option value table that we cannot modify to store extra data. 
I think we can store this value by creating a setting and storing the singular value for each case category in that setting. 
An example is the finance management setting here: https://github.com/compucorp/uk.co.compucorp.civiawards/blob/master/CRM/CiviAwards/Service/FinanceManagementSetting.php